.zuul.yml

```diff
@@ -19,6 +19,7 @@ browsers:
 server: ./test/support-server/index.js
 scripts:
   - /node_modules/jquery/dist/jquery.min.js
+  - /node_modules/jQuery-ajaxTransport-XDomainRequest/jquery.xdomainrequest.min.js
```

Comment:
this allows testing the jQuery plugin in IE9, this will be added to the changelog and migration guide